straxen/plugins/position_reconstruction.py

```diff
@@ -98,22 +99,31 @@ def compute(self, peaks):
         return result
 
     def _get_model_file_name(self):
-        config_file = f'file_{self.algorithm}'
-        model_file = self.config.get(config_file, "No file")
-        if model_file == 'No file':
+
+        config_file = f'{self.algorithm}_model'
+        model_from_config = self.config.get(config_file, 'No file')
+        if model_from_config == 'No file':
             raise ValueError(f'{__class__.__name__} should have {config_file} '
                              f'provided as an option.')
+        if isinstance(model_from_config, str) and os.path.exists(model_from_config):
+            # Allow direct path specification
+            return model_from_config
+        if model_from_config is None:
+            # Allow None to be specified (disables processing for given posrec)
+            return model_from_config
+
+        # Use CMT
+        model_file = straxen.get_NN_file(self.run_id, model_from_config)
         return model_file
 
-
 @export
 @strax.takes_config(
-    strax.Option("file_mlp",
-                 help="Name of saved MLP model file in the aux file data base."
-                      "The file contains both structure and weights. Set to "
-                      "None to skip the computation of this plugin.",
-                 default="xnt_mlp_wfsim_20201214.tar.gz"
-                 )
+    strax.Option('mlp_model',
+                 help='Neural network model.' 
+                      'If CMT, specify as (CMT_model, (mlp_NN_weights, ONLINE), True)))'

```

Comment:
A little too fast a copy pasta 😄 , the help is now the same for all of the plugins.

Reply:
yeah too much copy "pasta"... that sounds good